src/include/migraphx/operators.hpp

```diff
@@ -84,6 +84,7 @@
 #include <migraphx/op/mod.hpp>
 #include <migraphx/op/mul.hpp>
 #include <migraphx/op/multibroadcast.hpp>
+#include <migraphx/op/nearbyint.hpp>
```

Comment:
Do we still need round.hpp? Thanks.

Reply:
I was confused by these previous review comments [1](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/2244#discussion_r1349341859),  [2](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/2244#discussion_r1349341998) and I thought I have to provide a new operator for this. Also I was insecure about dropping the previous round operator because it was still used in quantize_linear.

Reply:
We do have a `round` operator in Onnx. It is just that internally it has to follow the logic of nearbyint(). So please do keep that operator by name. Perhaps it would be far simpler if the name `round` was left as it is, and the new operator name nearbyint() wasn't introduced at all. That is my preference. @pfultz2 & @umangyadav ?